### PR TITLE
Update Wizarr version to 3.4.2, use latest image in docker-compose.ym…

### PR DESCRIPTION
**Pull request by realashleybailey**
…l, and add a new logo file

- Version in `config.json` was updated from 2.2.0 to 3.4.2 to reflect the new Wizarr version.
- In `docker-compose.yml`, the image tag was changed from `2.2.0` to `latest` to use the latest Wizarr image.
- The old logo file `logo.jpg` was deleted, and a new logo file `logo.png` was added.

I am the admin of Wizarr for clarification

### CODE REVIEW

File: apps/wizarr/config.json [hunk 1]
```diff
@@ -6,12 +6,12 @@
   "exposable": true,
   "id": "wizarr",
   "tipi_version": 5,
-  "version": "2.2.0",
+  "version": "latest",
```
Comment (meienberger) [inline apps/wizarr/config.json:9]: We prefer to use fixed versions for the tag. It avoids unexpected breaking automatic updated (for example 2.2.0 to 3.4.2) Our scripts will automatically update the minor and patch versions

File: apps/wizarr/config.json [hunk 2]
```diff
@@ -6,12 +6,12 @@
...
   "categories": [
     "utilities"
   ],
-  "description": "Wizarr is a automatic user invitation system for Plex, Jellyfin and Emby. Create a unique link and share it to a user and they will automatically be invited to your Media Server! They will even be guided to download the clients and instructions on how to use your requests software!",
-  "short_desc": "Wizarr is an advanced user invitation and management system for Jellyfin, Plex, Emby etc.",
+  "description": "Wizarr is a automatic user invitation system for Plex and Jellyfin. Create a unique link and share it to a user and they will be invited to your Media Server after they complete there signup proccess! They can even be guided to download the clients and read instructions on how to use your media software!",
+  "short_desc": "Wizarr is a automatic user invitation system for Plex and Jellyfin.",
```
Comment (meienberger) [inline apps/wizarr/config.json:14]:
```suggestion
  "short_desc": "Wizarr is an automatic user invitation system for Plex and Jellyfin.",
```

File: apps/wizarr/docker-compose.yml [hunk 3]
```diff
@@ -3,15 +3,11 @@ version: "3.8"
 services:
   wizarr:
     container_name: wizarr
-    image: ghcr.io/wizarrrr/wizarr:2.2.0
+    image: ghcr.io/wizarrrr/wizarr:latest
```
Comment (meienberger) [inline apps/wizarr/docker-compose.yml:6]:
```suggestion
    image: ghcr.io/wizarrrr/wizarr:3.4.2
```

File: apps/wizarr/config.json (hunk 1 above)
Comment (meienberger) [inline apps/wizarr/config.json:9]:
```suggestion
  "version": "3.4.2",
```

**Review by meienberger — CHANGES_REQUESTED** [review]
Thank you for your PR! Have you checked the release notes of wizarr for the breaking v3? Does it work without any action from users to move from v2?

File: apps/wizarr/metadata/logo.png
Comment (meienberger) [inline apps/wizarr/metadata/logo.png:1]: This needs to be a `.jpg` file

**Comment by realashleybailey** [comment]
> Thank you for your PR! Have you checked the release notes of wizarr for the breaking v3? Does it work without any action from users to move from v2?

Yes, V2 can directly update to V3, I have yet to update the master README to reflect that the latest V3 can auto migrate now.

**Comment by meienberger** [comment]
> Yes, V2 can directly update to V3, I have yet to update the master README to reflect that the latest V3 can auto migrate now.

Didn't realise you were a Wizarr developer :) well I trust you in this case! Is the PR ready or do you have some work left?

**Comment by realashleybailey** [comment]
> > Yes, V2 can directly update to V3, I have yet to update the master README to reflect that the latest V3 can auto migrate now.
> 
> 
> 
> Didn't realise you were a Wizarr developer :) well I trust you in this case! Is the PR ready or do you have some work left?
> 
> 
> 
> 

Yes everything is ready I believe ☺️

**Pull request merged**